src/evidently/analyzers/classification_performance_analyzer.py

```diff
@@ -31,12 +31,18 @@ def calculate(self,
 
             # calculate quality metrics
             accuracy_score = metrics.accuracy_score(reference_data[target_column], reference_data[prediction_column])
-            avg_precision = metrics.precision_score(reference_data[target_column], reference_data[prediction_column],
-                                                    average='macro')
-            avg_recall = metrics.recall_score(reference_data[target_column], reference_data[prediction_column],
-                                              average='macro')
-            avg_f1 = metrics.f1_score(reference_data[target_column], reference_data[prediction_column],
-                                      average='macro')
+            if len(prediction_column) > 2:
```

Comment:
maybe will be better to set average='binary' or average='macro' depending on len(prediction_column) and set it explicitly in the same code for both cases?